pinot-core/src/test/java/org/apache/pinot/core/plan/maker/MetadataAndDictionaryAggregationPlanMakerTest.java

```diff
@@ -125,7 +126,7 @@ public void loadSegment()
     _upsertIndexSegment = ImmutableSegmentLoader.load(new File(INDEX_DIR, SEGMENT_NAME), ReadMode.heap);
     ((ImmutableSegmentImpl) _upsertIndexSegment).enableUpsert(
         new PartitionUpsertMetadataManager("testTable_REALTIME", 0, serverMetrics, null,
-            UpsertConfig.HashFunction.NONE), new ThreadSafeMutableRoaringBitmap());
+            UpsertConfig.HashFunction.NONE, Collections.emptyList()), new ThreadSafeMutableRoaringBitmap());
```

Comment:
(minor) Not introduced in this PR, but we should put a primary key column in the schema, and also include it here